src/settings-web/src/components/GeneralSettings.tsx

```diff
@@ -61,6 +61,7 @@ export class GeneralSettings extends React.Component <any, any> {
         { Object.keys(power_toys_enabled).map(
           (key) => {
             let enabled_value=power_toys_enabled[key];
+            let is_active=this.state.settings.powertoys[key].description !== 'This feature requires Windows 10, May 2019 Update';
```

Comment:
What about just checking if the description begins with "This feature requires"?